charts/agent/templates/_deployment-cluster-events-config.tpl

```diff
@@ -226,24 +251,127 @@ processors:
-          - set(attributes["observe_transform"]["facets"]["readyReplicas"], body["status"]["readyReplicas"])
+          - set(attributes["observe_transform"]["identifiers"]["statefulSetName"], body["metadata"]["name"])
+          - set(attributes["observe_transform"]["identifiers"]["statefulSetUid"], body["metadata"]["uid"])
           # spec
-          - set(attributes["observe_transform"]["facets"]["service"], body["spec"]["serviceName"])
+          - set(attributes["observe_transform"]["facets"]["serviceName"], body["spec"]["serviceName"])
           - set(attributes["observe_transform"]["facets"]["podManagementPolicy"], body["spec"]["podManagementPolicy"])
           - set(attributes["observe_transform"]["facets"]["desiredReplicas"], body["spec"]["replicas"])
           - set(attributes["observe_transform"]["facets"]["updateStrategy"], body["spec"]["updateStrategy"]["type"])
           - set(attributes["observe_transform"]["facets"]["partition"], body["spec"]["updateStrategy"]["rollingUpdate"]["partition"])
+          # status
+          - set(attributes["observe_transform"]["facets"]["currentReplicas"], body["status"]["currentReplicas"])
+          - set(attributes["observe_transform"]["facets"]["readyReplicas"], body["status"]["readyReplicas"])
+      - context: log
+        conditions:
+          - body["kind"] == "ConfigMap"
+        statements:
+          - set(attributes["observe_transform"]["identifiers"]["configMapName"], body["metadata"]["name"])
+          - set(attributes["observe_transform"]["identifiers"]["configMapUid"], body["metadata"]["uid"])
+          - set(attributes["observe_transform"]["facets"]["immutable"], body["immutable"])
+          - set(attributes["observe_transform"]["facets"]["data"], body["data"])
+      - context: log
+        conditions:
+          - body["kind"] == "Secret"
+        statements:
+          - set(attributes["observe_transform"]["identifiers"]["secretName"], body["metadata"]["name"])
+          - set(attributes["observe_transform"]["identifiers"]["secretUid"], body["metadata"]["uid"])
+          - set(attributes["observe_transform"]["facets"]["immutable"], body["immutable"])
+          - set(attributes["observe_transform"]["facets"]["data"], body["data"])
+          - set(attributes["observe_transform"]["facets"]["type"], body["type"])
+      - context: log
+        conditions:
+          - body["kind"] == "PersistentVolume"
+        statements:
+          - set(attributes["observe_transform"]["identifiers"]["persistentVolumeName"], body["metadata"]["name"])
+          - set(attributes["observe_transform"]["identifiers"]["persistentVolumeUid"], body["metadata"]["uid"])
+          # spec
+          - set(attributes["observe_transform"]["facets"]["capacity"], body["spec"]["capacity"]["storage"])
+          - set(attributes["observe_transform"]["facets"]["persistentVolumeReclaimPolicy"], body["spec"]["persistentVolumeReclaimPolicy"])
```

Comment:
for these facets, why prefix with "persistentVolume" 

ie why not just "volumeReclaimPolicy" and "volumeClass"


same for PersistentVolumeClaim below